components/constants/pref_names.h

```diff
@@ -61,6 +61,7 @@ inline constexpr char kShieldsStatsBadgeVisible[] =
 inline constexpr char kAdControlType[] = "brave.ad_default";
 inline constexpr char kGoogleLoginControlType[] = "brave.google_login_default";
 inline constexpr char kWebTorrentEnabled[] = "brave.webtorrent_enabled";
+// Deprecated
```

Comment:
I don't think this comment is needed here. right now it looks like everything below is deprecated.

Reply:
@goodov How about adding right side of the definition?

```suggestion
inline constexpr char kWebTorrentEnabled[] = "brave.webtorrent_enabled"; // Deprecated
```

Reply:
Webtorrent is OK; it's the one below that is deprecated

Reply:
Yes yes, I just wanted to show it with example :laughing: